Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -1,5 +1,3 @@
-require "os/mac/pathname"
```

Comment:
?

Reply:
I think this line was an anachronism, before `extend/os` existed. My opinion…
`extend/os/mac/hardware/cpu.rb` should not require `os/mac/pathname.rb`
`extend/pathname.rb` should require `extend/os/pathname.rb` which requires `extend/os/mac/extend/pathname.rb`

Reply:
Question: is there a way to detect _anachronisms_ like this? That is, something is required but is not, in fact, necessary.